packages/rn-tester/RNTester-visionOS/ReactRootView.swift

```diff
@@ -0,0 +1,16 @@
+import SwiftUI
+import React
+import React_RCTAppDelegate
+
+struct ReactRootView: UIViewControllerRepresentable {
+  var moduleName: String
+  var initialProps: [AnyHashable: Any]?
+  
+  func makeUIViewController(context: Context) -> UIViewController {
+    RCTReactViewController(moduleName: moduleName, initProps: initialProps)
+  }
+  
+  func updateUIViewController(_ uiViewController: UIViewController, context: Context) {
+    // noop
+  }
+}
```

Comment:
Try to ship this inside of RCTAppDelegate

Reply:
Unfortunately, not doable due to the dependants of `RCTAppDelegate` not defining modules.

Reply:
Created a separate package to store this